documentation/en/api-v1-unstable-methods.md

```diff
@@ -1899,232 +1990,1184 @@ Response:
+  "blockNumber": "0x5",
+  "from": "0x5cbeecf99d3fdb3f25e309cc264f240bb0664031",
+  "to": "0x5cbeecf99d3fdb3f25e309cc264f240bb0664031",
+  "root": "0x37690cfec6c1bf4c3b9288c7a5d783e98731e90b0a4c177c2a374c7a9427355e",
+  "status": "0x5",
+  "contractAddress": "0x5cbeecf99d3fdb3f25e309cc264f240bb0664031",
+  "cumulativeGasUsed": "0x5",
+  "gasUsed": "0x5",
+  "effectiveGasPrice": "0x0",
+  "logsBloom": "0x07",
+  "logs": [
+    {
+      "address": "0x5cbeecf99d3fdb3f25e309cc264f240bb0664031",
+      "data": "0x07",
+      "topics": [
+        "0x07"
+      ],
+      "removed": true,
+      "logIndex": "0x5",
+      "transactionIndex": "0x5",
+      "transactionHash": "0x37690cfec6c1bf4c3b9288c7a5d783e98731e90b0a4c177c2a374c7a9427355e",
+      "blockHash": "0x37690cfec6c1bf4c3b9288c7a5d783e98731e90b0a4c177c2a374c7a9427355e",
+      "blockNumber": "0x5"
+    }
+  ],
+  "type": "0x5"
 }
 ```
 
-### ClientRemoveImport
-ClientRemoveImport removes file import
+### EthMaxPriorityFeePerGas
 
 
-Perms: admin
+Perms: read
 
-Inputs:
-```json
-[
-  50
-]
-```
+Inputs: `null`
 
-Response: `{}`
+Response: `"0x0"`
 
-### ClientRestartDataTransfer
-ClientRestartDataTransfer attempts to restart a data transfer with the given transfer ID and other peer
+### EthNewBlockFilter
+Installs a persistent filter to notify when a new block arrives.
 
 
 Perms: write
 
-Inputs:
+Inputs: `null`
+
+Response:
 ```json
 [
-  3,
-  "12D3KooWGzxzKZYveHXtpG6AsrUJBcWxHBFS2HsEoGTxrMLvKXtf",
-  true
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  92,
+  190,
+  236,
+  1,
+  35,
+  69,
+  103,
+  63,
+  37,
+  227,
+  9,
+  204,
+  38,
+  79,
+  36,
+  11,
+  176,
+  102,
+  64,
+  49
 ]
 ```
 
-Response: `{}`
-
-### ClientRetrieve
-ClientRetrieve initiates the retrieval of a file, as specified in the order.
+### EthNewFilter
+Installs a persistent filter based on given filter spec.
 
 
-Perms: admin
+Perms: write
 
 Inputs:
 ```json
 [
   {
-    "Root": {
-      "/": "bafy2bzacea3wsdh6y3a36tb3skempjoxqpuyompjbmfeyf34fi3uy6uue42v4"
-    },
-    "Piece": null,
-    "DataSelector": "Links/21/Hash/Links/42/Hash",
-    "Size": 42,
-    "Total": "0",
-    "UnsealPrice": "0",
-    "PaymentInterval": 42,
-    "PaymentIntervalIncrease": 42,
-    "Client": "f01234",
-    "Miner": "f01234",
-    "MinerPeer": {
-      "Address": "f01234",
-      "ID": "12D3KooWGzxzKZYveHXtpG6AsrUJBcWxHBFS2HsEoGTxrMLvKXtf",
-      "PieceCID": null
-    },
-    "RemoteStore": "00000000-0000-0000-0000-000000000000"
+    "fromBlock": "2301220",
+    "address": [
+      "0x5cbeecf99d3fdb3f25e309cc264f240bb0664031"
+    ],
+    "topics": null
   }
 ]
 ```
 
 Response:
 ```json
-{
-  "DealID": 5
-}
+[
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  0,
+  92,
+  190,
+  236,
+  1,
+  35,
+  69,
+  103,
+  63,
+  37,
+  227,
+  9,
+  204,
+  38,
+  79,
+  36,
+  11,
+  176,
+  102,
+  64,
+  49
+]
```

Comment:
Some of those responses don't look quite right, are we sure this isn't what's sent over the wire?

Reply:
We are missing a doc example for the EthFilterID type.